runtime/src/lib.rs

```diff
@@ -74,3 +74,50 @@ impl<B: BlockT> ParachainBlockData<B> {
 		&self.extrinsics
 	}
 }
+
+
+pub mod validation_function_params {
```

Comment:
Locating this module directly under `runtime` is kind of an accident; I was just looking for a place where its imports would require the fewest new dependencies. if there's a better place to put it, I'm happy to move it.

Reply:
Good idea; done in https://github.com/paritytech/cumulus/pull/70/commits/91f12d9ecf055a02099f9f8ca2b457355a6f5c06.